root/defaults/nginx/nginx.conf.sample

```diff
@@ -70,24 +73,9 @@ http {
     # Sets the path, format, and configuration for a buffered log write.
     access_log /config/log/nginx/access.log;
 
-    ### Linuxserver.io Defaults
-
-    client_body_buffer_size 128k; # default: client_body_buffer_size 8k|16k;
-    #keepalive_timeout 75s; # default: keepalive_timeout 75s;
-    large_client_header_buffers 4 16k; # default: large_client_header_buffers 4 8k;
-    send_timeout 5m; # default: send_timeout 60s;
-    #server_name_in_redirect off; # default: server_name_in_redirect off;
-    #server_names_hash_bucket_size 64; # default: server_names_hash_bucket_size 32|64|128;
-    #tcp_nodelay on; # default: tcp_nodelay on;
-    types_hash_max_size 2048; # default: types_hash_max_size 1024;
-    variables_hash_max_size 2048; # default: variables_hash_max_size 1024;
```

Comment:
All of the lines that were commented out were using the default values nginx ships with. The ones that were not commented out were all increased from their default values, but after researching each of them it does not seem necessary to do so. If an individual app (downstream) requires these values changed, the changes should be applied to the individual app, not the base.